crates/swc_common/src/syntax_pos/hygiene.rs

```diff
@@ -289,19 +177,13 @@ impl Default for HygieneData {
 impl HygieneData {
     pub(crate) fn new() -> Self {
         HygieneData {
-            marks: vec![MarkData {
-                parent: Mark::root(),
-                // If the root is opaque, then loops searching for an opaque mark
-                // will automatically stop after reaching it.
-                is_builtin: true,
-            }],
             syntax_contexts: vec![SyntaxContextData {
                 outer_mark: Mark::root(),
                 prev_ctxt: SyntaxContext(0),
                 opaque: SyntaxContext(0),
-                opaque_and_semitransparent: SyntaxContext(0),
             }],
             markings: HashMap::default(),
+            mark_count: 1,
```

Comment:
If you search through the codebase `Mark::fresh()` always pass one mark which mostly is `Mark::root` which means it will at least > `Mark (0)`, so we start with 1, it has the same effect, make the `Mark::fresh()` create mark consistently great than `Mark::root()`